platform/darwin/src/MGLStyle.h

```diff
@@ -318,6 +318,27 @@ MGL_EXPORT
  */
 - (void)removeSource:(MGLSource *)source;
 
+/**
+ Removes a source from the current style.
+ 
+ @note Source identifiers are not guaranteed to exist across styles or different
+ versions of the same style. Applications that use this API must first set the
+ style URL to an explicitly versioned style using a convenience method like
+ `+[MGLStyle outdoorsStyleURLWithVersion:]`, `MGLMapView`’s “Style URL”
+ inspectable in Interface Builder, or a manually constructed `NSURL`. This
+ approach also avoids source identifer name changes that will occur in the default
+ style’s sources over time.
+ 
+ @param source The source to remove from the current style.
+ @param outError Upon return, if an error has occurred, a pointer to an `NSError`
+ object describing the error. Pass in `NULL` to ignore any error.
```

Comment:
Nit: `nil`

Reply:
Not that is makes a difference in practice, but since `nil` is used with objects (`id`), and `NSError**` isn't a pointer-to-object (rather a pointer-to-pointer-to-object), it should be `NULL` in this case.